lms/templates/navigation-edx.html

```diff
@@ -101,44 +103,28 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
       % endif
 
     % else:
-    <nav aria-label="Main" class="nav-main">
-      <ul class="left nav-global">
-        <%block name="navigation_global_links">
-            <li class="nav-global-01">
-              <a href="${marketing_link('HOW_IT_WORKS')}">${_("How it Works")}</a>
-            </li>
-            <li class="nav-global-02">
-              <a href="${marketing_link('COURSES')}">${_("Find Courses")}</a>
-            </li>
-            <li class="nav-global-03">
-              <a href="${marketing_link('SCHOOLS')}">${_("Schools & Partners")}</a>
-            </li>
-        </%block>
-      </ul>
-    </nav>
-
     <nav aria-label="Account" class="nav-account-management">
       <div class="right nav-courseware">
+        <div class="nav-courseware-01">
+          % if not settings.FEATURES['DISABLE_LOGIN_BUTTON']:
+              % if course and settings.FEATURES.get('RESTRICT_ENROLL_BY_REG_METHOD') and course.enrollment_domain:
+              <a class="cta cta-login" href="${reverse('course-specific-login', args=[course.id.to_deprecated_string()])}${login_query()}">${_("Sign in")}</a>
+              % else:
+              <a class="cta cta-login" href="/login${login_query()}">${_("Sign in")}</a>
+              % endif
+          % endif
+        </div>
         % if not settings.FEATURES['DISABLE_LOGIN_BUTTON']:
           % if course and settings.FEATURES.get('RESTRICT_ENROLL_BY_REG_METHOD') and course.enrollment_domain:
-          <div class="nav-courseware-01">
-            <a class="cta cta-register" href="${reverse('course-specific-register', args=[course.id.to_deprecated_string()])}">${_("Register")}</a>
+          <div class="nav-courseware-02">
+            <a class="cta cta-register nav-courseware-button" href="${reverse('course-specific-register', args=[course.id.to_deprecated_string()])}">${_("Register")}</a>
           </div>
           % else:
-          <div class="nav-courseware-01">
-            <a class="cta cta-register" href="/register">${_("Register")}</a>
+          <div class="nav-courseware-02">
+            <a class="cta cta-register nav-courseware-button" href="/register">${_("Register")}</a>
           </div>
           % endif
         % endif
```

Comment:
These two if structures aren't the same...one puts a blank div the other doesn't....is this on purpose?